irohad/torii/processor/impl/transaction_processor_impl.cpp

```diff
@@ -97,17 +96,11 @@ namespace iroha {
       model::TransactionResponse response;
       response.tx_hash = hash(*transaction).to_string();
       response.current_status =
-          model::TransactionResponse::Status::STATELESS_VALIDATION_FAILED;
+          model::TransactionResponse::Status::STATELESS_VALIDATION_SUCCESS;
```

Comment:
Probably this STATELESS_VALIDATION_SUCCESS is no longer relevant here, since we can answer it after builder. What about signatures, where are they checked?  Maybe add todo  here.